tests/suite/test_virtual_server_dos.py

```diff
@@ -133,6 +141,7 @@ def fin():
         delete_dos_policy(kube_apis.custom_objects, pol_name, test_namespace)
         delete_dos_logconf(kube_apis.custom_objects, log_name, test_namespace)
         delete_dos_protected(kube_apis.custom_objects, protected_name, test_namespace)
+        clean_good_bad_clients()
         # delete_items_from_yaml(kube_apis, src_webapp_yaml, test_namespace)
```

Comment:
@pasmant  more commented code here, can you please have a look?

Reply:
Done